### DISCUSSION TITLE
Question: are you able configure Renovate to not update existing PullRequest?

### DISCUSSION THREAD

**Discussion by JaMertKhlr** [renovatebot/renovate#32878] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure

### Please tell us more about your question or problem

Hello Renovate Team,

We have set up RenovateBot to update our NuGet packages, and we have encountered a use case that we are struggling to resolve. Currently, all dependency updates are grouped together into a single pull request per base branch, which is our preferred setup. We need time to test the changes and cannot always review the pull requests promptly. Therefore, we would prefer that Renovate does not update the pull request. We have defined the base PR branch to always look the same: 
`"branchName": "renovate/{{baseBranch}}/DependencyUpdates". `

The challenge we face is that Renovate continues to commit new updates to the branch as long as it remains unmerged or open. We would like Renovate to either ignore new updates while keeping the branch up-to-date or add a timestamp to the created branch so we can differentiate the commits.

Is there a configuration option that supports this use case, or is this outside the intended functionality of Renovate?

Thank you for your assistance!

Best regards!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment]
Check out `rebaseWhen`

**Reply by rarkins** [reply]
I think the PR is getting updated due to newer versions being released, and not due to rebasing

**Comment by rarkins** [comment, marked as answer]
Try adding the label `stop-updating` to the PR. If you're unable to use that particular label, you can configure it using `stopUpdatingLabel` in Renovate config

**Reply by JaMertKhlr** [reply]
Thank you very much for your response, and I apologize for my (very) late reply. Using the stop-updating label worked perfectly for our case.

I just want to check if something like this would also be possible: Is there a way to configure Renovate to always create a new branch for the PR using a prefix or suffix? For instance, when Renovate finds new updates and notices that a previously created PR is still open, it could create a new branch and a new PR, perhaps by adding a variable like a timestamp. Is this within the intended functionality of Renovate?

**Reply by RahulGautamSingh** [reply]
No.

You can use `branchPrefix` to change the prefix (default `renovate/`) used in the branch names.  But, If a previous PR is open for an update another PR for the same update will not be opened.

**Reply by JaMertKhlr** [reply]
Then we are fine with using the stop-updating label. Thank you very much!
